pytorch_toolkit/ote/init_venv.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
```

Comment:
One more `init_venv.sh`?

Reply:
Yes! It is required for internal `ote` tests.

Reply:
It looks weird and confusing...  Generally, each template have own `venv` and tests should be run inside it but we have one more `venv` inside `ote` without any relation to templates - maybe move to the tests folder at least?

Reply:
I understand that is not obvious, but from the tests point of view `ote` is just one more domain that should be tested. 
The reason was as follows:
* we need to test `ote`
* it is a good idea to make `ote` tests together with all other tests for all domains
* for testing `ote` we need to have a special virtual environment -- so, `init_venv.sh` should be created for it
* `ote` is a folder in the same hierarchy level as other domains

So it is a convenient to have `ote` for tests as just one more domain.

Why it is not good?

Reply:
In my opinion, the tests should not impact on the structure of the main code, but when we see something like `create venv in root dir of package` means that we should create this `venv` but finally it will be figured out that it's only for internal tests of some functionality (which is not used by everyone who want to add own repo).

Reply:
let's do two lines required to run tests https://github.com/openvinotoolkit/training_extensions/tree/develop/pytorch_toolkit#tests, no need more venvs